OptionSelector/CategoryMain.m

```diff
@@ -0,0 +1,39 @@
+//
+//  CategoryMain.m
+//  OptionSelector
+//
+//  Created by Charles Kang on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CategoryMain.h"
+#import "CategorySetup.h"
+
+@interface CategoryMain ()
```

Comment:
It's good practice to include at least one comment for each class describing what it does or what data it stores (if it's just a data storage class). Even if it seems obvious to you now, it might not in a month (and it might not to your colleagues).